core/ir/riscv64/instr_create_api.h.in

```diff
@@ -40,14 +40,14 @@
 /** @{ */ /* doxygen start group */
 
 /* FIXME i#3544: Check and implement all INSTR_CREATE_ and XINST_CREATE_ macros. For now
- * all unimplemented instructions use OP_hint which is effectively a no-op. */
+ * all unimplemented instructions use OP_c_nop which is effectively a no-op. */
```

Comment:
Curious why we needed to make this change from OP_hint to OP_c_nop.

Reply:
> fixup opcode references which aren't valid anymore

Oh, probably this.

Reply:
Exactly :)